frontend/src/app/core/services/http.service.ts

```diff
@@ -0,0 +1,58 @@
+import { HttpClient, HttpErrorResponse } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable, throwError } from 'rxjs';
+import { catchError } from 'rxjs/operators';
+
+@Injectable({
+    providedIn: 'root',
+})
+export class HttpService {
+    // eslint-disable-next-line no-empty-function
+    constructor(private httpClient: HttpClient, private baseUrl: string) {
```

Comment:
Can you load baseUrl from some config file/variable?
Or register HttpService globally, so it will be set automatically

Reply:
fixed